examples/react-native/README.md

```diff
@@ -0,0 +1,83 @@
+# Getting Started
+
+## Setup React Native Environment
+
+Follow the React Native docs for [adding the required dependencies](https://reactnative.dev/docs/environment-setup#installing-dependencies) with **macOS** as the Development Environment, select either **Android** or **iOS** as the target OS.
```

Comment:
Even though you mentioned `React Native docs`, the link didn't direct to that tab so I still spent some time finding `installing-dependencies`, so maybe emphasis it like "React Native docs(not Expo CLI)" ? 😂